src/components/AssetsTable/index.jsx

```diff
@@ -68,16 +74,25 @@ export class AssetsTable extends React.Component {
       },
     };
 
+    this.deleteActions = [assetActions.DELETE_ASSET_FAILURE, assetActions.DELETE_ASSET_SUCCESS];
+
     this.trashcanRefs = {};
     this.statusAlertRef = {};
 
-    this.onDeleteClick = this.onDeleteClick.bind(this);
-    this.deleteAsset = this.deleteAsset.bind(this);
     this.addSupplementalTableElements = this.addSupplementalTableElements.bind(this);
     this.closeModal = this.closeModal.bind(this);
     this.closeStatusAlert = this.closeStatusAlert.bind(this);
-    this.renderStatusAlert = this.renderStatusAlert.bind(this);
+    this.deleteAsset = this.deleteAsset.bind(this);
     this.onCopyButtonClick = this.onCopyButtonClick.bind(this);
+    this.onDeleteClick = this.onDeleteClick.bind(this);
+    this.renderStatusAlert = this.renderStatusAlert.bind(this);
+    this.updateUploadSuccessCount = this.updateUploadSuccessCount.bind(this);
+  }
+
+
+  componentWillReceiveProps(nextProps) {
+    const { assetsStatus } = nextProps;
+    this.getStatusAlertFields(assetsStatus);
   }
```

Comment:
Will update StatusAlertFields if the incoming props denote an update is needed for rendering the StatusAlert